_episodes/03-create.md

```diff
@@ -207,15 +207,15 @@ draft.txt
 > 3.  When might you want to create a file this way?
 >
 > > ## Solution
-> > 1.  The touch command generates a new file called 'my_file.txt' in
+> > 1.  The touch command generates a new file called `my_file.txt` in
 > >     your home directory.  If you are in your home directory, you
-> >     can observe this newly generated file by typing 'ls' at the 
-> >     command line prompt.  'my_file.txt' can also be viewed in your
+> >     can observe this newly generated file by typing `ls` at the 
+> >     command line prompt.  `my_file.txt` can also be viewed in your
```

Comment:
Is this inconsistent with #778? The name of a file is not a runnable command. Same issue again below.

Reply:
@gcapes I created https://github.com/swcarpentry/lesson-example/pull/201 to try to have some ground rules. Could you let me know what do you thing?